appIos/appIos/NewsfeedApp.swift

```diff
@@ -18,6 +18,7 @@ struct NewsfeedApp: App {
  Applies the correct app theme based on the device's configuration.
  */
 private struct ThemeView: View {
+   // @Environment(.colorScheme) var ea
```

Comment:
remove this - added for understanding

Reply:
Looks like this is still here so we can remove when ready - I'll unresolve so that we don't forget 👍

Reply:
Removed now